src/dockerfiles/Makefile

```diff
@@ -18,8 +18,11 @@ build-system-metric:
 build-base-connector:
 	docker build . -t aqueducthq/base_connector -f connectors/base.dockerfile --no-cache
 
-build-connectors: build-bigquery-connector build-mysql-connector build-postgres-connector \
-	build-s3-connector build-snowflake-connector build-sqlserver-connector
+build-connectors: build-athena-connector build-bigquery-connector build-mysql-connector \
+	build-postgres-connector build-s3-connector build-snowflake-connector build-sqlserver-connector
+
+build-athena-connector:
```

Comment:
@cw75 You will need to create this image and push it tagged as `0.0.12`

Reply:
@saurav-c since Athena wasn't part of the 0.0.12 release, I don't have to do this right?

Reply:
ah right, we can wait until 0.0.13